run_sysbench.sh

```diff
@@ -0,0 +1,30 @@
+pgip=127.0.0.1
+port=5433
+user=yugabyte
+db=yugabyte
+run_threads=64
+table_size=1000000
+time=120
+prefx='YB_1Mil_'
+
+load_workload() {
+  echo "LOADING"
+  time sysbench oltp_insert --tables=1 --table-size=$table_size --range_key_partitioning=true --serial_cache_size=1000 --db-driver=pgsql --pgsql-host=$pgip --pgsql-port=$port --pgsql-user=$user --pgsql-db=$db --threads=1 prepare > $prefx_load.txt
```

Comment:
Let's not output to a file but print to stdout instead.
It's not going to be obvious to users that we are writing to a file.

Reply:
Done.